app/src/androidTest/java/org/mozilla/focus/utils/AndroidTestUtils.java

```diff
@@ -133,6 +131,18 @@ public static void tapSettingButton() {
         onView(allOf(withId(R.id.menu_preferences), isDisplayed())).perform(click());
     }
 
+    public static void tapPrivateButtonInMenu() {
+        onView(allOf(withId(R.id.btn_private_browsing), isDisplayed())).perform(click());
+    }
+
+    public static void tapPrivateButtonInTabtray() {
```

Comment:
can we also change the re-use id here in production code?
This will make id-based ui testing like hell